experiments/AMIP/components/ocean/slab_ocean_init.jl

```diff
@@ -96,7 +96,7 @@ function ocean_init(
 
     evolving_switch = evolving ? FT(1) : FT(0)
     params =
-        OceanSlabParameters(FT(20), FT(1500.0), FT(800.0), FT(271.0), FT(1e-5), FT(1e-5), FT(0.38), evolving_switch)
+        OceanSlabParameters(FT(20), FT(1500.0), FT(800.0), FT(271.0), FT(5e-4), FT(5e-4), FT(0.38), evolving_switch)
```

Comment:
instead of passing all of these values directly into the function, could we assign them to variables beforehand then pass them in so we can see what each one is? (as is done for the parameters in bucket_init.jl)

Reply:
I followed the `@kwdef` method of the eisenman sea ice to clean up the parameter specification here and in the prescribed sea ice files.